semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtils.java

```diff
@@ -431,13 +431,17 @@ public static URL getNormalizedUrl(String url) throws MalformedURLException {
       }
     }
 
+    if (httpUrlFromRawAttributes != null && isAbsoluteUrl(httpUrlFromRawAttributes)) {
```

Comment:
It same logic, right? It is refactored?

Reply:
I think we missed some conditions in previous way

Reply:
which condition?

Reply:
yes got confused. Looks just refactoring